master/usage/ip-pool-manipulation.md

```diff
@@ -0,0 +1,69 @@
+---
+title: IP Pool Manipulation
+---
+The 
+[IP Pools]({{site.baseurl}}/{{page.version}}/reference/calicoctl/resources/ippool)
+that Calico starts up with can be manipulated and changed.  This document
+describes how to do that using the `calicoctl` utility.
+
+## Retrieving IP Pool information
+
+The current IP Pools can be retrieved with the following commmand.
+
+```shell
+$ calicoctl get ipPool
+CIDR
+192.168.0.0/16
+
+```
+
+To retrieve the information about a specific IP Pool you must specify the pool
```

Comment:
This is a bit misleading - specifying the pool CIDR isn't necessary to see its specific information, it just filters down to that one pool. For example, you can print `calicoctl get ippool -o yaml` to get the specifics of all IP pools.